src/pybind/mgr/dashboard/controllers/nfsganesha.py

```diff
@@ -162,11 +164,10 @@ def create(self, path, cluster_id, daemons, pseudo, tag, access_type,
                  },
                  responses={200: EXPORT_SCHEMA})
     def get(self, cluster_id, export_id):
-        export_id = int(export_id)
-        ganesha_conf = GaneshaConf.instance(cluster_id)
-        if not ganesha_conf.has_export(export_id):
-            raise cherrypy.HTTPError(404)
-        return ganesha_conf.get_export(export_id).to_dict()
+        # Get export by pseudo path?
+        #export_mgr = mgr.remote('nfs', 'fetch_nfs_export_obj')
+        #return export_mgr._get_export_dict(cluster_id, pseudo)
+        return mgr.remote('nfs', 'export_get', cluster_id, export_id)
```

Comment:
That's an interesting question. RESTful APIs usually identify resources (URIs) with unique IDs (same as DB primary keys, etc): `/api/nfs/export/0`, `/api/nfs/export/1`, etc. Using the pseudo-path as ID involves encoding the a path in the URI and that usually looks awful (replacing `/` with `%2F`: `/api/nfs/export/%2Fthis%2Fis%2Fa%2Fpseudo%2Fpath`.

Additionally, renames are usually challenging as it'd involve `DELETE old_uri` + `PUT new_uri`, instead of a regular PUT/PATCH to the ID-based URI.

As discussed, if possible, it'd be better to keep the export ID-based API for the Dashboard.